playwright-tests/tests/communities.spec.js

```diff
@@ -91,6 +91,52 @@ test.describe("Wallet is connected", () => {
       true
     );
   });
+  test("should create a new community", async ({ page }) => {
+    await page.goto("/devgovgigs.near/widget/app?page=communities");
```

Comment:
Should this be `/devhub.near/widget/app/?page=communities`?

Reply:
Indeed it should be. I've changed it for the other community tests too now. Actually that also required some changes in the tests. Thank you for finding this.